tests/pytest/test_binary_value.py

```diff
@@ -8,6 +8,32 @@
 TRUNCATION_MATCH = r"\d+-bit value requested, truncating value"
 
 
+def test_init_zero_length():
+    bin1 = BinaryValue(value=0, n_bits=0, binaryRepresentation=BinaryRepresentation.UNSIGNED)
+    assert bin1._str == ""
+    assert bin1.binstr == ""
+    assert bin1.integer == 0
+
+    bin2 = BinaryValue(value=0, n_bits=0,
+                       binaryRepresentation=BinaryRepresentation.TWOS_COMPLEMENT)
+    assert bin2._str == ""
+    assert bin2.binstr == ""
+    assert bin2.integer == 0
+
+    bin3 = BinaryValue(value=0, n_bits=0,
+                       binaryRepresentation=BinaryRepresentation.SIGNED_MAGNITUDE)
+    assert bin3._str == ""
+    assert bin3.binstr == ""
+    assert bin3.integer == 0
+
+    # Whatever value is set to a zero bit long BinaryValue, it should read 0
+    bin4 = BinaryValue(value=10, n_bits=0,
```

Comment:
This one should warn

Reply:
It does. I just changed the test to catch this. 
I would expect that I get an error when setting:
bin4 <= 5

But no..